modules/wallpapers/CMakeLists.txt

```diff
@@ -8,3 +8,4 @@ add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/ReactOS_CirclesGold_1.jpg DESTINATI
 add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/ReactOS_CirclesGold_2.jpg DESTINATION reactos/Web/Wallpaper FOR all)
 add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/ReactOS_Hexagons.jpg DESTINATION reactos/Web/Wallpaper FOR all)
 add_subdirectory(Angelus)
+add_subdirectory(Silhouette)
```

Comment:
![image](https://user-images.githubusercontent.com/578406/53513630-ce4bd780-3ad6-11e9-99b9-28c29ee2c092.png)

This might be problematic.

Reply:
how so?